### PR TITLE
DS-3561  Automated Monthly Patches - Aug23

### PR DESCRIPTION
**Pull request by loyaltynz-svc-cicd**
# Automated Monthly Patches
**Base Image:** docker.loyaltydevops.co.nz/service_base_ruby:3.2.2
**Command used:** # bundle update --patch
**Jenkins build:** https://jenkins.loyaltydevops.co.nz/job/TechOps/job/cron-jobs/job/MonthlyPatching/job/bundle_update/714/

## Changes:
- rack 2.2.8 (was 2.2.7)
- rexml 3.2.6 (was 3.2.5)
- addressable 2.8.5 (was 2.8.4)
- rspec-mocks 3.12.6 (was 3.12.5)

### CODE REVIEW

**Comment by ildomar-grings** [comment]
retest this please

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +18,15 @@
 - Replace gem `le` with `r7insight` - [DS-3525](https://loyaltynz.atlassian.net/browse/DS-3525)
   - Callers must include  the `r7insight` gem instead of `le` gem in their `Gemfile`
 
+## 3.2.10
```
Comment (davidoram) [inline CHANGELOG.md:21]: Is this supposed to be here?

**Review by davidoram — COMMENTED** [review]
Small question around teh chnagelog

File: CHANGELOG.md (hunk 1 above)
Comment (ildomar-grings) [reply CHANGELOG.md:21]: My gosh. I totally forgot that. 
Fixing it now.

Comment (ildomar-grings) [reply CHANGELOG.md]: Done.

**Review by davidoram — APPROVED** [review]
GTG

**Pull request merged**